package/src/skia/TextBlob.ts

```diff
@@ -0,0 +1,52 @@
+import type { SkJSIInstance } from "./JsiInstance";
+import type { IFont } from "./Font/Font";
+import type { IRSXform } from "./RSXform";
+
+export type ITextBlob = SkJSIInstance<"TextBlob">;
+
+export interface TextBlobFactory {
+  /**
+   * Return a TextBlob with a single run of text.
+   *
+   * It uses the default character-to-glyph mapping from the typeface in the font.
+   * It does not perform typeface fallback for characters not found in the Typeface.
+   * It does not perform kerning or other complex shaping; glyphs are positioned based on their
+   * default advances.
+   * @param str
+   * @param font
+   */
+  MakeFromText(str: string, font: IFont): ITextBlob;
+  /**
+   * Return a TextBlob with a single run of text.
+   *
+   * It does not perform typeface fallback for characters not found in the Typeface.
```

Comment:
I'm sure there are use cases for this, but I likely would not use this because of the limitations listed here in my use case specifically 🤔